server/app/views/admin/programs/ProgramApplicationListView.java

```diff
@@ -161,7 +174,27 @@ private FormTag renderSearchForm(
                 .setValue(filterParams.search().orElse(""))
                 .setLabelText("Search by name, email, or application ID")
                 .getInputTag()
-                .withClasses(Styles.W_FULL, Styles.MT_4),
+                .withClasses(Styles.W_FULL, Styles.MT_4))
+        .condWith(
+            allApplicationStatuses.size() > 0,
+            new SelectWithLabel()
+                .setFieldName(APPLICATION_STATUS_PARAM)
+                .setLabelText("Application status")
+                .setValue(filterParams.selectedApplicationStatus().orElse(""))
+                .setOptions(
+                    ImmutableMap.<String, String>builder()
+                        .put("Any application status", "")
+                        .put(
+                            "Only applications without a status",
```

Comment:
What do you think about putting a "divider" selection to distinguish between these special options and the real one?

Just thinking "-------"